account_lock_to_date/models/account_move.py

```diff
@@ -18,6 +18,8 @@ def _check_lock_to_dates(self):
         for move in self:
             advisor_lock_to_date = move.company_id.fiscalyear_lock_to_date
             user_lock_to_date = move.company_id.period_lock_to_date
+            if self.env.context.get("bypass_account_lock_to_date"):
+                continue
```

Comment:
This bypass could be outside the for loop. You don't need to skip each line individually.

Reply:
I have made the requested change by moving the bypass check outside the for loop, thx for review